BE/OwnershipAndControl/Executives.rdf

```diff
@@ -422,7 +426,7 @@
 	
 	<owl:Class rdf:about="&fibo-be-oac-exec;CorporateOfficer">
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;Signatory"/>
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;OrganizationMember"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-org-org;OrganizationMember"/>
 		<rdfs:label>corporate officer</rdfs:label>
 		<skos:definition>high-level management executive of a corporation or other organization, hired by the board of directors or the business owner(s), charged with certain operational responsibilities</skos:definition>
```

Comment:
> It seems wrong to use the word "executive" in this definition since Executive is a separate class which this does not subtype.

The intent is to look at a number of these relationships and integrate them into the situational pattern appropriately, including adding executive as a parent class.

> Making AgentInRole disjoint with IndependentParty is a showstopper that unnecessarily prevents useful multiple classification. Why can't Person Pete also be classified as an Owner?

This has been in the ontology for at least a couple of years, probably longer.  It has been important for testing data quality for one of my customers, where we have been integrating content from multiple sources to address problems related to identity resolution and other things.  For the use cases we've tested, it works, and it allowed integration with BFO for one government user, which makes a similar distinction.  Not all FIBO users will need to integrate with BFO-based ontologies, of course, but making this distinction is critical if they do.